x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v2_windows_rare_metadata_process.json

```diff
@@ -0,0 +1,38 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "This is a new refactored job which works on ECS compatible events across multiple indices. Security: Windows - Looks for anomalous access to the metadata service by an unusual process. The metadata service may be targeted in order to harvest credentials or user data scripts containing secrets.",
+  "groups": [
+    "security",
+    "endpoint",
+    "event-log",
+    "process",
+    "sysmon",
+    "windows",
+    "winlogbeat"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"process.name\"",
+        "function": "rare",
+        "by_field_name": "process.name"
+      }
+    ],
+    "influencers": [
+      "agent.id",
+      "host.name",
+      "user.name"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "32mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "created_by": "ml-module-security-windows"
```

Comment:
Is there a reason why we don't have the custom_urls to the security app for this job?

Reply:
Same reason as https://github.com/elastic/kibana/pull/71323 -  These fields, in events from cloud workloads, do not present super well in conventional network / host pages and the primary workflow will be in the detections and timeline pages after these get turned into detection alerts. A hostname is not an entity in cloud environments so we'd need to rework the hosts page to use the instance GUID.